puppet/puppet-manifests/personalize-desktop.pp

```diff
@@ -0,0 +1,35 @@
+#node default { }
+
+node 'dev-box' {
+
+	exec { 'make dev user default':
+		command => '/bin/sed -i "s/vagrant/dev/g" /etc/lightdm/lightdm.conf',
+	}
+	
+	exec { 'add osgp wallpaper':
+		command => '/bin/cp -f /home/vagrant/repos/Config/vagrant/OSGP-solar.png /usr/share/backgrounds/warty-final-ubuntu.png',
```

Comment:
Why overwrite the default ubuntu png? Isn't it possible to just copy osgp-solar.png and set it as default wallpaper via gsettings? (see for example [this Ask Ubuntu page](http://askubuntu.com/questions/66914/how-to-change-desktop-background-from-command-line-in-unity)